app/controllers/employees_controller.rb

```diff
@@ -1,14 +1,29 @@
 class EmployeesController < ApplicationController
+  
   before_action :require_employee_logged_in!, only: :show
 
-  def index; end 
+  def index
+    redirect_to edit_employee_registration_path
+  end 
 
   def show
     @employee = Employee.find_by(id: params[:id])
```

Comment:
If you are using id to find user (data) from table use this syntax.
**Employee.find(params[:id])**